readme.md

```diff
@@ -140,7 +171,15 @@ Instantiate an instructor named 'Andy' and call its `receivePresent` method with
 
 ```js
 // Your code starts here...
+function Instructor(name) {
+  this.name = name
+  this.receivePresent = function(gift){
+    console.log(name + " promptly drops the " gift  + " on the floor.")
```

Comment:
This line should reference `this.name` since it's a property on the instance, not a local variable.

Additionally, best practice would be to define methods on the prototype, to prevent making multiple copies of the same method.

```js
function Instructor(name){
  this.name = name;
}
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}

var andy = new Instructor("Andy");
andy.receivePresent("Resin Laying Deer Figurine, Gold");
```